.travis.yml

```diff
@@ -3,5 +3,10 @@ osx_image: xcode11.4
 branches:
   only:
   - master
+install:
+  - ./install_swiftlint.sh
+  - ./install_swiftformat.sh
 script:
-- set -o pipefail && swift test
+  - swiftlint lint --quiet
```

Comment:
quiet still outputs reasons for failure, right?

Reply:
Yeah, it just suppresses the `Linting /some/file` messages that it prints for every file by default